### DISCUSSION TITLE
How to handle users' commands inside a group chat?

### DISCUSSION THREAD

**Discussion by jialeee17** [def-studio/telegraph#562] [Q&A]
My bot currently only works with user chats (1 to 1). How can the bot work inside a group chat where there're multiple users. Is it possible that the bot is able to response to different users' command inside a group chat? 

In order to do this, I'm trying to find out which user the message was belong to and get the user's data. I noticed that there's a "memberInfo('user_id')" method provided by "DefStudio\Telegraph\Models\TelegraphChat" model. To use the method, I have to provide a "user_id" parameter. To get the "user_id", I noticed that the User DTO (https://defstudio.github.io/telegraph/features/dto#user) might be able to help, but I have no idea how to use it. Can anyone help? Thanks in advance!

**Comment by fabio-ivona** [comment]
From your webhook you are able to access to the `$message` property, where you can find the user that sent the message
